<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1749</width>
    <height>911</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>999999</width>
        <height>999999</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Main">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QPushButton" name="zoomInButton">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSlider" name="zoomSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <widget class="QScrollArea" name="panelScrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>248</width>
             <height>752</height>
            </rect>
           </property>
           <widget class="QWidget" name="widgetParent" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>251</width>
              <height>101</height>
             </rect>
            </property>
           </widget>
           <widget class="QFrame" name="frameParent">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>290</y>
              <width>120</width>
              <height>80</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="debugButton">
          <property name="text">
           <string>Debug</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="FlightMapWidget" name="ShowMapWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="zoomOutButton">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="centerForWp">
          <property name="text">
           <string>Центр</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Edit">
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Команда</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>param1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>param2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>param3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>param4</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Широта</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Долгота</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Высота</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Удалить</string>
           </property>
           <property name="font">
            <font>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дистанция</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Вверх</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Вниз</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Углы</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Град %</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="updButton">
          <property name="text">
           <string>Обновить таблицу</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1749</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Параметры</string>
    </property>
    <addaction name="action_spif"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть полётное задание</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="action_spif">
   <property name="text">
    <string>Задать путь к spif</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FlightMapWidget</class>
   <extends>QWidget</extends>
   <header location="global">flightmapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
